sig-observability/concepts-and-designs/how-to-add-new-metric-to-kubesphere.md

```diff
@@ -0,0 +1,61 @@
+# Why we need to add metrics to kubesphere
+
+Metrics can provide insight into the behavior and health of our system. They represent the raw material used by monitoring system to build a holistic view of kubesphere, automate responses to changes, and alert human beings when required.
```

Comment:
Maybe we should add ：

- Description of the metrics scraping port and path 

- An instrumentation overview link https://prometheus.io/docs/practices/instrumentation/

- more naming conversion instructions links like:
https://prometheus.io/docs/practices/naming/ 
https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels

- And also some metrics type instructions links like:
https://prometheus.io/docs/concepts/metric_types/

- Intro to the client library 
https://prometheus.io/docs/instrumenting/clientlibs/